README.md

```diff
@@ -80,6 +80,14 @@ microk8s can be restarted later with the snap command
 snap enable microk8s
 ```
 
+### Removing microk8s
+
+To remove microk8s we recommend you first disable all addons and stop all hosted pods. microk8s.reset can assist with that.
```

Comment:
Recommend changing this sentence to "Before removing microk8s, use `microk8s.reset` to stop all running pods."